src/Microsoft.AspNetCore.SignalR.Specification.Tests/Microsoft.AspNetCore.SignalR.Specification.Tests.csproj

```diff
@@ -4,7 +4,7 @@
     <Description>Tests for users to verify their own implementations of SignalR types</Description>
     <!-- Temporarily suppress packing this project until we can fix issues in its dependencies. -->
     <IsPackable>false</IsPackable>
-    <DeveloperBuildTestTfms>netcoreapp2.2</DeveloperBuildTestTfms>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
     <StandardTestTfms>$(DeveloperBuildTestTfms)</StandardTestTfms>
```

Comment:
I think you can remove the next 6 lines as well as IsPackable=false (above).

Also, just to be clear, we're explicitly not supporting netstandard2.0 for these tests, right?